WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java

```diff
@@ -383,6 +424,7 @@ public static FirstMatcher first() {
         return new FirstMatcher();
     }
 
+
```

Comment:
Probably don't need the extra whitespace here?

Reply:
Done